opflexagent/utils/bridge_managers/vpp_manager.py

```diff
@@ -0,0 +1,143 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from neutron.plugins.ml2.drivers.openvswitch.agent.common import constants
+from opflexagent.utils.bridge_managers import bridge_manager_base
+from opflexagent.utils.bridge_managers import trunk_skeleton
+from oslo_config import cfg
+from oslo_log import log as logging
+from vpplib.VPPApi import VPPApi
```

Comment:
This should be part of the requirements

Reply:
ok.. I need to add all vpp specific requirements.. will do.. thanks..

Reply:
As I mentioned, we are still discussing on whether to package vpplib ourselves or to have vpp team do it. But UT can work regardless, since we are stubbing this import for now.